client/components/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+class App extends React.Component {
+  constructor() {
```

Comment:
Make sure to pass in props to the constructor and super once you get to that point! :)